inst/remindoutputfiles.Rmd

```diff
@@ -0,0 +1,175 @@
+      x = "Time",
+      y = "Value",
+      color = "EDGE Scenario"
+    ) +
+    theme_minimal() +
+    theme(legend.position = "bottom")
+  
+  # Print the plot (or save it if you want to store the plots)
+  print(p)
+  
+  # Optionally, save the plot to a file
+  # ggsave(paste0("plot_", teEs, ".png"), plot = p, width = 8, height = 6)
+}
+
+```
+
+
+## Energy efficiency per energy service technology
+```{r fe2es, echo=FALSE}
+unique_teEs <- unique(f35_fe2es$all_teEs)
+
+# Loop through each unique value of all_teEs and create a plot
+for (teEs in unique_teEs) {
+  
+  # Filter data for the current technology/service (all_teEs)
+  filtered_data <- f35_fe2es[all_teEs == teEs, ]
+  
+  # Create the ggplot for the current teEs
+  p <- ggplot(filtered_data, aes(x = tall, y = value, color = EDGE_scenario)) +
+    geom_line() +
+    facet_wrap(~ all_regi, scales = axisScaling) +  # Facet by region/country
+    labs(
+      title = paste("Energy efficiency for:", teEs),
+      x = "Time",
+      y = "Value",
+      color = "EDGE Scenario"
+    ) +
+    theme_minimal() +
+    theme(legend.position = "bottom")
+  
+  # Print the plot (or save it if you want to store the plots)
+  print(p)
+  
+  # Optionally, save the plot to a file
+  # ggsave(paste0("plot_", teEs, ".png"), plot = p, width = 8, height = 6)
+}
+
+```
+
+
+## Capital cost per energy service technology
+```{r capital cost, echo=FALSE}
+unique_teEs <- unique(f35_esCapCost$all_teEs)
+
+# Loop through each unique value of all_teEs and create a plot
+for (teEs in unique_teEs) {
+  
+  # Filter data for the current technology/service (all_teEs)
+  filtered_data <- f35_esCapCost[all_teEs == teEs, ]
+  
+  # Create the ggplot for the current teEs
+  p <- ggplot(filtered_data, aes(x = tall, y = value, color = EDGE_scenario)) +
+    geom_line() +
+    facet_wrap(~ all_regi, scales = axisScaling) +  # Facet by region/country
+    labs(
+      title = paste("Capital cost for:", teEs),
+      x = "Time",
+      y = "Value",
+      color = "EDGE Scenario"
+    ) +
+    theme_minimal() +
+    theme(legend.position = "bottom")
+  
+  # Print the plot (or save it if you want to store the plots)
+  print(p)
+  
+  # Optionally, save the plot to a file
+  # ggsave(paste0("plot_", teEs, ".png"), plot = p, width = 8, height = 6)
+}
+
+```
+
+## Final energy demand per energy service technology
+```{r demByTech, echo=FALSE}
+unique_teEs <- unique(f35_demByTech$all_teEs)
+
+# Loop through each unique value of all_teEs and create a plot
+for (teEs in unique_teEs) {
+  
+  # Filter data for the current technology/service (all_teEs)
+  filtered_data <- f35_demByTech[all_teEs == teEs]
+  
+  # Create the ggplot for the current teEs
+  p <- ggplot(filtered_data, aes(x = tall, y = value, color = EDGE_scenario)) +
+    geom_line() +
+    facet_wrap(~ all_regi, scales = axisScaling) +  # Facet by region/country
+    labs(
+      title = paste("Capital cost for:", teEs),
```

Comment:
this could be changed to "Final energy demand for: "  - the wrong title confused me for a short moment :-)

Reply:
Thanks for spotting :) I think I will remove it from the package and store it in our helper script repo